build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -186,9 +258,75 @@
         Overwrite="True"
     />
   </Target>
+
+  <!-- The condition below is to work around a bug in xbuild which attempts to batch
+       even if there are no _MonoCrossRuntime items and the Copy task fails
+  -->
+  <Target Name="_ConfigureCrossRuntimes"
```

Comment:
Cross and AOT are different things. Cross means the **tool** used to produce AOT code for specific **target** as in "a cross-compiler which uses AOT process to generate a binary for target architecture/OS X"